model/channel.go

```diff
@@ -128,6 +128,12 @@ type ChannelSearchOpts struct {
 	PerPage                *int
 }
 
+type ChannelMemberCountByGroup struct {
+	GroupId                     string `json:"group_id"`
+	ChannelMemberCount          int64  `json:"channel_member_count"`
```

Comment:
@fm2munsh, should we have `db:"-" json:"channel_member_count"` instead, since it is not a db table field?

Reply:
my assumption is that since the struct itself does not a table associated with it theres no point specifying that the column does not exist either

Reply:
groupChannelJoin (and others) is not directly mapped to a table either and specifies that some fields are marshaled from table fields. My assumption was that for every struct that is marshaled from of a table query, one needs to (or is it recommended?) to specify if its fields are built/constructed at query time or they directly map to table columns.  A clear pattern to follow would def. help (me at least).

Reply:
cool - will apply this change 👍